automation/ansible/roles/setup_jmeter_client/tasks/main.yml

```diff
@@ -0,0 +1,59 @@
+---
+- name: Install jmeter client packages.
+  become: yes
+  become_user: root
+  ansible.builtin.package:
+    name: "{{ jmeter_packages }}"
+    state: present
+
+- name: Prepare JMeter client directory
+  shell: |
+    if [ ! -d "{{ jmeter_remote_bin_dir_full_path }}" ]; then
```

Comment:
You can change this to use ansible file module. This was just a easy way for me to test out my changes.

Reply:
Made the changes and I will be checking in soon.